Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -1105,7 +1115,7 @@ public virtual void Impact(Thing hitThing)
                 }
 
                 foreach (var thing in suppressThings)
-                    ApplySuppression(thing);
+                    ApplySuppression(thing, 3f);
```

Comment:
I can either keep this value at `3f` or reduce it, as I am unsure of how balanced it is.

Reply:
It's usually best to make "magic constants" use constant variables at the top of the class (for examples, look at the top of ProjectileCE).  

So it would read `ApplySuppression(thing, ExplosionSuppression);`